packages/select/src/Select/Select.spec.tsx

```diff
@@ -936,14 +936,6 @@ describe('packages/select', () => {
       expect(isError).toBeTruthy();
       expect(getErrorMessage()).toBeInTheDocument();
     });
-
-    test('error message is not present if "errorMessage" is empty', () => {
```

Comment:
Should we test the default error message though?

Reply:
I added the default message at the form field level